server_admin/topics/authentication/webauthn.adoc

```diff
@@ -0,0 +1,197 @@
+[[_webauthn]]
+
+=== W3C Web Authentication (WebAuthn)
+
+{project_name} provides the limited support for https://www.w3.org/TR/webauthn/[W3C Web Authentication (WebAuthn)]. {project_name} works as WebAuthn's https://www.w3.org/TR/webauthn/#rp-operations[Relying Party (RP)].
+
+IMPORTANT: Please note that this WebAuthn support is still in development phase and not yet fully completed so that it is recommended to use this WebAuthn support in experimental use. Moreover, there is a possibility that this support's specification and user interfaces will be changed in the future.
+
+NOTE: Whether WebAuthn's operations succeeds depends on a user's WebAuthn supporting authenticator, browser and platform. If you use this WebAuthn support, please clarify to what extent those entities supports WebAuthn specification.
+
+The major restriction is as follows :
+
+- A user can register only one WebAuthn supporting Authenticator (called WebAuthn authenticator here)
+- On registering the WebAuthn authenticator, its https://www.w3.org/TR/webauthn/#attestation-statement[Attestation Statement] is not verified
+- Only Two-Factor Authentication (2FA) scenario is supported
+
+==== Setup
+
+The setup procedure of WebAuthn support for 2FA is the following :
+
+Enable User Registration::
+
+An administrator carries out the following operations on `Admin Console` :
+
+- open `Realm Settings -> Login` tab
+- set `User registration` switch ON and push `Save` button
+
+Enable Webauthn Authenticator Registration::
+
+An administrator carries out the following operations on `Admin Console` :
+
+- open `Authentication -> Required Actions` tab
+- push `Register` button
+- select `Webauthn Register` as `Required Action`
+- mark `Enabled` and `Default Action` checkbox
+
+Enable 2FA by Webauthn Authenticator::
+
+An administrator carries out the following operations on Admin Console :
+
+- open `Authentication -> Flows` tab
+- copy and create the new browser flow
+- add `WebAuthn Authenticator` execution below `User Password Form` execution
+- set `OPTIONAL` to `WebAuthn Authenticator` execution's requirement
+- open `Authentication -> Bindings` tab
+- select the created new browser flow as `Browser Flow`
+- push `Save` button
+
+==== Register WebAuthn Authenticator
+
+There are two types of registering a WebAuthn authenticator, for a user who has not yet registered their account on {project_name}, and for a user who has already registered their account on {project_name}.
+
+For the user to identify their registered WebAuthn authenticator afterwards, the user can enter the text as their label.
+
+User without their account::
+
+A user carries out the following operations :
+
+- open the login form
```

Comment:
Add THE before "Register' link and 'Register' button...

Reply:
Thank you. I'll fix this sentence as you suggested.